api/outdated/settings.py

```diff
@@ -202,3 +203,13 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
 
 # Variables used only in testing
 VALIDATE_REMOTES = True
+
+# health checks
+WATCHMAN_CHECKS = env.list(
+    "WATCHMAN_CHECKS",
+    default=(
+        "watchman.checks.caches",
+        "watchman.checks.databases",
```

Comment:
do you know what this does to check the database?

Reply:
```py
@check
def _check_database(database):
    connection = connections[database]
    with connection.cursor() as cursor:
        cursor.execute("SELECT 1")
    return {database: {"ok": True}}
```

Reply:
nice, that's what i was expecting